config/webpack.prod.config.js

```diff
@@ -103,12 +115,12 @@ module.exports = merge(commonConfig, {
             loader: 'postcss-loader',
             options: {
               postcssOptions: {
-                plugins: [
-                  PostCssAutoprefixerPlugin(),
-                  PostCssRTLCSS(),
-                  CssNano(),
-                  PostCssCustomMediaCSS(),
-                ],
+                plugins: process.env.USE_PURGECSS // If USE_PURGECSS is set we append it,
+                  ? [...postCSSLoaderPlugins,
+                    purgecss({
+                      content: ['./**/*.html', './**/*.js', './**/*.jsx', './**/*.ts', './**/*.tsx'],
+                    })]
+                  : postCSSLoaderPlugins,
```

Comment:
I think it would improve readability (and future extensibility) to use the `extraPlugins` pattern already used in this file.  For example, you could do something like:

```javascript
const extraPostCssPlugins = [];
if (process.env.USE_PURGESS !== 'false') {
  extraPostCssPlugins.push(purgecss({
    content: ['./**/*.html', './**/*.js', './**/*.jsx', './**/*.ts', './**/*.tsx'],
  }));
}
```

Then:

```javascript
                plugins: [
                  PostCssAutoprefixerPlugin(),
                  PostCssRTLCSS(),
                  CssNano(),
                  PostCssCustomMediaCSS(),
                  ...extraPostCssPlugins,
                ],
```

Reply:
Make sense but just to double confirm we are on the same page, would you prefer to by deafult this would be enabled or disabled?